modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/prepare/ddl/AbstractDdlCommand.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.query.calcite.prepare.ddl;
+
+/**
+ * Abstract ddl command.
+ */
+public abstract class AbstractDdlCommand implements DdlCommand {
+    /** Table name. */
+    private String tblName;
+
+    /**
+     * Schema name upon which this statement has been issued - <b>not</b> the name of the schema where this new table will be created.
```

Comment:
which schema will the table be created in?
Maybe 'defaultSchema' or 'commanCurrentSchema' or smth like will be better name for the field?

Reply:
got it, change param, append additional comment and test.